src/containers/pages/InterventionPlan/UpdatePlan/index.tsx

```diff
@@ -43,13 +43,15 @@ const UpdatePlan = (props: RouteComponentProps<RouteParams> & UpdatePlanProps) =
   if (!planIdentifier) {
     return null; /** we should make this into a better error page */
   }
+  const apiService = new service(OPENSRP_PLANS);
 
   /** async function to load the data */
   async function loadData() {
     try {
       setLoading(plan === null); // only set loading when there are no plans
       const planFromAPI = await apiService.read(planIdentifier);
-      fetchPlan(planFromAPI);
+      const currentPlan = Array.isArray(planFromAPI) ? planFromAPI[0] : planFromAPI;
+      fetchPlan(currentPlan);
```

Comment:
The data we are getting from API is a `list` but we were using it as an `object` thus when filtering using the `planIdentifier` it could not be found.